Solutions/MaturityModelForEventLogManagementM2131/Workbooks/MaturityModelForEventLogManagement_M2131.json

```diff
@@ -7,18 +7,33 @@
         "version": "KqlParameterItem/1.0",
```

Comment:
Yes, this is desired state. imAuthentication is Azure Sentinel Information Model (ASIM), there are multiple versions deployed which errors. In the customer space they wouldn't have duplicates so this would work as expected. The other log sources are related to VPNs which we don't have in the demo environment so they'll error out. This is the expected state for log sources that aren't available and serve as an assessment to recommend that a customer onboard these controls.